functorch/_src/eager_transforms.py

```diff
@@ -314,7 +314,7 @@ def jacrev(func: Callable, argnums: Union[int, Tuple[int]] = 0, *, has_aux=False
         returns the Jacobian of :attr:`func` with respect to the arg(s) at
         :attr:`argnums`. If ``has_aux is True``, then the returned function
         instead returns a ``(jacobian, aux)`` tuple where ``jacobian``
-        is the Jacobian and ``aux`` is auxiliary objects returned by ``func``.
+        is the Jacobian and ``aux`` is auxiliary objects returned by :attr:`func`.
```

Comment:
Why the change? I thought :attr: is for attributes on objects -- func is an argument to the function.

Reply:
> Why the change?

To make the docs a bit more uniform. It says 
```
        Returns a function that takes in the same inputs as :attr:`func` and
        returns the Jacobian of :attr:`func` with respect to the arg(s) at
        :attr:`argnums`. If ``has_aux is True``, then the returned function
        instead returns a ``(jacobian, aux)`` tuple where ``jacobian``
        is the Jacobian and ``aux`` is auxiliary objects returned by :attr:`func`.
```

So, the last `:attr:'func'` is the same as the one from 
```
        Returns a function that takes in the same inputs as :attr:`func` and
        returns the Jacobian of :attr:`func` with respect to the arg(s) at
```

Reply:
In general I think we are using :attr: wrong, but it doesn't have any negative effects so this is fine :)

Reply:
Yes, I agree that in rendered docs :attr:`func` does not do what I was thinking. Anyway, there is a room of improvements for the docstrings as well.